concepts.md

```diff
@@ -93,8 +94,8 @@ To store additional information about the device, the application uses device tw
 
 | Name    	| Position     	| Description                                                                                                                                 	|
 |---------	|--------------	|---------------------------------------------------------------------------------------------------------------------------------------------	|
-| Purpose 	| ``tags.purpose`` 	| Field that contains the Device Type value.<br>_**note**: this tag may  be used to target deployment manifests for the IoT Edge_ 	|
 | Environment 	| ``tags.env`` 	| Field that contains the Device Environment value.<br>_**note**: this tag may be used to target deployment manifests for the IoT Edge_ 	|
+| Type 	| ``tags.type`` 	| Field that contains The type of the IoT Edge device.<br>_**note**: this tag may be used to target deployment manifests for the IoT Edge_ 	|
```

Comment:
Could we specify that the expected value is ``LoRa Network Server`` for IoT Edge LoRaWAN LNS ?

Reply:
I also included the value `Other`